packages/admin/cms-admin/src/pages/useSaveConflictQuery.tsx

```diff
@@ -20,13 +21,18 @@ export function useSaveConflictQuery<TData, TVariables>(
     const { resolveHasConflict, skip, ...restOptions } = options;
     const checkConflict = async () => {
         if (skip) return false;
-        const { data, error } = await client.query({
-            query,
-            fetchPolicy: "no-cache",
-            ...restOptions,
-        });
-        if (error) return false;
-        return resolveHasConflict(data);
+        try {
+            const { data, error } = await client.query({
+                query,
+                fetchPolicy: "no-cache",
+                context: LocalErrorScopeApolloContext,
```

Comment:
@nsams this prevents the global error handling in all usages of `useSaveConflictQuery`, which I think is okay. What do you think?

Reply:
Also, should we treat this as breaking change?

Reply:
:+1:

it's a bugfix :)